cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -43,26 +43,30 @@ enum class Radix : int32_t {
 
 namespace detail {
     // helper function to negate strongly typed scale_type
+    __host__ __device__
```

Comment:
Use `CUDA_HOST_DEVICE_CALLABLE` so that this file can still be compiled in host-only code: https://github.com/rapidsai/cudf/blob/branch-0.13/cpp/include/cudf/types.hpp#L23

Reply:
Done.